test262/main.js

```diff
@@ -118,6 +136,50 @@
       }
     }
 
+    const performanceChartStartDateTime = DateTime.fromISO("2021-07-04");
```

Comment:
Let's make it more clear this is a constant, i.e. uppercase and move further up (e.g. after line 21) - and also add a comment:

```js
// This is when we started running the tests on Idan's self-hosted runner. Before that,
// durations varied a lot across runs. See https://github.com/SerenityOS/serenity/pull/7718.
const PERFORMANCE_CHART_START_DATE_TIME = DateTime.fromISO("2021-07-04");
```